nessus-agent/windows/init.sls

```diff
@@ -6,45 +6,21 @@
 # In addiont, the sls file requires that the salt winrepo has been
 # configured with a `nessus agent` pkg definition
 
-{%- if nessus.log_rotation_strategy %}
-Add Log Rotation Strategy:
+{%- for log_file,log_rotation in nessus.log_config.items() %}
+  {%- if log_rotation %}
+    {%- for rotation_param,rotation_value in log_rotation.items() %}
+      {%- if rotation_value %}
```

Comment:
this if block ought not be needed once the log_config default is an empty dict. from there, it's user error if they set a log_config and also pass an empty value for any particular setting.

Reply:
Agree.  I'll remove.